docs/en/advanced_tutorials/logging.md

```diff
@@ -1,3 +1,295 @@
+runner = Runner(
+    model=ToyModel(),
+    work_dir='tmp_dir',
+    train_dataloader=train_dataloader,
+    train_cfg=dict(by_epoch=True, max_epochs=1),
+    optim_wrapper=dict(optimizer=dict(type='SGD', lr=0.01))
+)
+runner.train()
+```
+
+```
+08/21 02:58:41 - mmengine - INFO - Epoch(train) [1][10/25]  lr: 1.0000e-02  eta: 0:00:00  time: 0.0019  data_time: 0.0004  loss1: 0.8381  loss2: 0.9007  loss: 1.7388
+08/21 02:58:41 - mmengine - INFO - Epoch(train) [1][20/25]  lr: 1.0000e-02  eta: 0:00:00  time: 0.0029  data_time: 0.0010  loss1: 0.1978  loss2: 0.4312  loss: 0.6290
+```
+
+LogProcessor will output the log as the following statistic method:
+
+- The prefix of the log:
+  - epoch mode(`by_epoch=True`): `Epoch(train) [{current_epoch}/{current_iteration}]/{dataloader_length}`
+  - iteration mode(`by_epoch=False`): `Iter(train) [{current_iteration}/{max_iteration}]`)
+- Learning rate(`lr`): The learing rate of the last iteration.
+- Time:
+  - `time`: The averaged time for infernce of the last `window_size` iterations.
+  - `data_time`: The averaged time for loading data of the last `window_size` iterations.
+  - `eta`: The estimated time of arrival to finish the training.
+- Loss: The averaged loss output by model of the last `window_size` iterations.
+
+```{note}
+`window_size=10` by default.
+
+The significant digits(`num_digits`) of the log is 4 by default.
+
+Output the value of all custom logsthe at last iteration by default.
+```
+
+```{warnning}
+log_processor output the epoch based log by default(`by_epoch=True`). To get a expected log matched with the `train_cfg`, we should set the same value for `by_epoch` in `train_cfg` and `log_processor`.
+```
+
+Based on the rules above, the sample code will statistic the average value of the `loss1` and `loss2` every 10 iterations.
+
+If we want to statistic the global average value of `loss1`, we can set `custom_cfg` like this:
+
+```python
+runner = Runner(
+    model=ToyModel(),
+    work_dir='tmp_dir',
+    train_dataloader=train_dataloader,
+    train_cfg=dict(by_epoch=True, max_epochs=1),
+    optim_wrapper=dict(optimizer=dict(type='SGD', lr=0.01)),
+    log_processor=dict(
+        custom_cfg=[
+            dict(data_src='loss1',  # original loss name：loss1
+                 method_name='mean',  # statistic method：mean
+                 window_size='global')])  # window_size：global
+)
+runner.train()
+```
+
+```
+08/21 02:58:49 - mmengine - INFO - Epoch(train) [1][10/25]  lr: 1.0000e-02  eta: 0:00:00  time: 0.0026  data_time: 0.0007  loss1: 0.7381  loss2: 0.8446  loss: 1.5827
+08/21 02:58:49 - mmengine - INFO - Epoch(train) [1][20/25]  lr: 1.0000e-02  eta: 0:00:00  time: 0.0030  data_time: 0.0012  loss1: 0.4521  loss2: 0.3939  loss: 0.5600
+```
+
+`data_src` means the original loss name, `method_name` means the statistic method, `window_size` means the window size of the statistic method. Since we want to statistic the global average value of `loss1`, we set `window_size` to `global`.
```

Comment:
```suggestion
`data_src` means the original loss name, `method_name` means the statistic method, `window_size` means the window size of the statistic method. Since we want to count the global average value of `loss1`, we set `window_size` to `global`.
```

Reply:
Seems more below